File: accidentes/FicheroAccidentes-Java-IvanRC/src/main/java/dto/lesividad/LesividadDto.java
package dto.lesividad;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Root(name = "lesividad")
public class LesividadDto {
    @Element(name = "id_lesividad")
    String id; // 13
    @Element(name = "lesividad")
    String lesividad; // 14
}

File: accidentes/FicheroAccidentes-Kotlin-IvanRC/src/main/kotlin/dto/lesividad/LesividadDto.kt
package dto.lesividad

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "lesividad")
data class LesividadDto (
    @field:Element(name = "id_lesividad")
    @param:Element(name = "id_lesividad")
    val id: String, // 13
    @field:Element(name = "lesividad")
    @param:Element(name = "lesividad")
    val lesividad: String // 14
)
